getPlaylistfromFiles.py

```diff
@@ -0,0 +1,70 @@
+import spotipy
+import spotipy.oauth2 as oauth2
+import json
+import time
+
+def readFromFile(filename) :
+	listOfLinks2 = []
+	with open("playlistLinks") as f:
+		line = f.readline()
+		while line:
+			listOfLinks2.append(line[:-1])
+			line = f.readline()
+	f.close()
+	return listOfLinks2
+
+
+
+#DO NOT TOUCH - DO NOT LOOK - PRIVATE KEY
+credentials = oauth2.SpotifyClientCredentials(''    ,'')
+token = credentials.get_access_token()
+sp = spotipy.Spotify(auth=token)
+
+#read from file where you specified your playlists
+listOfLinks = readFromFile('playlistLinks')
+#open file for writing results
+file = open("tracksFeatures", "w")
```

Comment:
File is one of python built-ins, please use another name
https://docs.python.org/2/library/functions.html